simp_le.py

```diff
@@ -243,8 +266,9 @@ class IOPlugin(object):
     - for `cert`: certificate, an instance of `OpenSSL.crypto.X509`
     - for `chain`: certificate chain, a list of `OpenSSL.crypto.X509` instances
     """
+    Data.__new__.__defaults__ = (None,) * 4
```

Comment:
Also, please keep PRs scope limited. This kind of change should go into a different PR.